src/components/ContentWidgets/ToDoList/TodoListEditor.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { WidgetEditorProps } from '../../ContentMapping/ContentMapping';
+import { TodoItem, TodoList } from './TodoList';
+import "./TodoStyle.css";
+
+// an input box for adding a new todo item or checking/unchecking
+// existing todo items.
+//
+// Last Modified
+// Max Zhou
+// July 6, 2019
+//
+export const TodoListEditor: React.FC<WidgetEditorProps> = ({
+  editedContent,
+  setEditedContentOnChange
+}) => {
+	// get the list
```

Comment:
unnecessary comment. The code is self documenting already, we can see it gets the current list.